lib/block-supports/layout.php

```diff
@@ -28,15 +28,16 @@ function gutenberg_register_layout_support( $block_type ) {
 /**
  * Generates the CSS corresponding to the provided layout.
  *
- * @param string  $selector                      CSS selector.
- * @param array   $layout                        Layout object. The one that is passed has already checked the existence of default block layout.
- * @param boolean $has_block_gap_support         Whether the theme has support for the block gap.
- * @param string  $gap_value                     The block gap value to apply.
- * @param boolean $should_skip_gap_serialization Whether to skip applying the user-defined value set in the editor.
+ * @param string     $selector                      CSS selector.
+ * @param array      $layout                        Layout object. The one that is passed has already checked the existence of default block layout.
+ * @param array|null $padding                       Padding applied to the current block.
+ * @param boolean    $has_block_gap_support         Whether the theme has support for the block gap.
+ * @param string     $gap_value                     The block gap value to apply.
+ * @param boolean    $should_skip_gap_serialization Whether to skip applying the user-defined value set in the editor.
  *
  * @return string                                CSS style.
  */
-function gutenberg_get_layout_style( $selector, $layout, $has_block_gap_support = false, $gap_value = null, $should_skip_gap_serialization = false ) {
+function gutenberg_get_layout_style( $selector, $layout, $padding, $has_block_gap_support = false, $gap_value = null, $should_skip_gap_serialization = false ) {
```

Comment:
Not related to this PR, but we might want to start using a single array of arguments for this function. I know BuddyPress has done similar deprecations in the past. So we could use a similar method.

I think we should follow the "two or fewer" recommendation from the Clean Code when it comes to function arguments.

Reply:
👍  The single array of arguments potentially just plasters over an underlying issue though of a method trying to do too much - would definitely be good to look at this method and see if a single method with all those arguments is the best approach for sure.